radicle-cli/src/git.rs

```diff
@@ -27,6 +28,30 @@ pub const CONFIG_GPG_FORMAT: &str = "gpg.format";
 pub const CONFIG_GPG_SSH_PROGRAM: &str = "gpg.ssh.program";
 pub const CONFIG_GPG_SSH_ALLOWED_SIGNERS: &str = "gpg.ssh.allowedSignersFile";
 
+/// Git `rev` identifier that it is used
+/// in the radicle cli in order to store and
+/// support the short sha1 without ambiguity
```

Comment:
```suggestion
/// Git revision parameter. Supports extended SHA-1 syntax.
```

Reply:
YEah I'm not good to write docs sorry :)